schemas/responses/optimzationResult.yaml

```diff
@@ -0,0 +1,24 @@
+OptimizationResult:
```

Comment:
There is a typo in this file's name.  `optimzationResult` -> `optimizationResult`